hazelcast/src/main/java/com/hazelcast/concurrent/atomiclong/AtomicLongService.java

```diff
@@ -147,4 +165,99 @@ private void clearPartitionReplica(int partitionId) {
             }
         }
     }
+
+    @Override
+    public Runnable prepareMergeRunnable() {
+        IPartitionService partitionService = nodeEngine.getPartitionService();
+        Map<Integer, List<AtomicLongContainer>> containerMap = createHashMap(partitionService.getPartitionCount());
+        Address thisAddress = nodeEngine.getClusterService().getThisAddress();
+
+        for (Map.Entry<String, AtomicLongContainer> entry : containers.entrySet()) {
+            AtomicLongContainer container = entry.getValue();
+            if (!(getMergePolicy(container) instanceof DiscardMergePolicy)) {
+                String name = entry.getKey();
+                int partitionId = partitionService.getPartitionId(StringPartitioningStrategy.getPartitionKey(name));
+                if (thisAddress.equals(partitionService.getPartitionOwner(partitionId))) {
```

Comment:
Minor: there's `partitionService.getPartition(partitionId).isLocal()` for this as well.

Reply:
I guess I'll change this in all PRs then.